src/components/Header.js

```diff
@@ -42,8 +38,8 @@ const Header = () => {
                     <Link to="/discount-page" className="header-link">
                         Акції
                     </Link>
-                    <ButtonComponet onClick={openModal} buttonName='Вхід'/>
-                    <div onClick={openBasket} className="header-basket">
+                    <ButtonComponent onClick={openModal} buttonName='Вхід'/>
```

Comment:
`onClick={() => setIsModalOpen(true)}`

Reply:
If you have one simple action - no need to create separated function. So check other clicks.